src/operator/concatAll.ts

```diff
@@ -12,6 +11,6 @@ import {MergeAllOperator} from './mergeAll-support';
  *
  * @returns {Observable} an observable of values merged from the incoming observables.
  */
-export function concatAll<T>(): Observable<T> {
+export function concatAll<T>(): T {
```

Comment:
`concatAll`/`mergeAll` all assume that you're already dealing with `Observable<Observable<T>>`, so calling `concatAll` actually returns `T` since `T` is `Observable<S>`